### ISSUE TITLE
Cannot get user leverage at bybit (ExchangeNotAvailable)

### ISSUE THREAD

**Issue by tradercgt** [ccxt/ccxt#6781]
I get the error **ExchangeNotAvailable** probably because the request is sent to the wrong URL: `https://api.bybit.comleverage/user/leverage`

- OS: Ubuntu 18.04
- Programming Language version: node v13.12.0
- CCXT version: 1.26.7

```
'use strict'

const ccxt = require('ccxt')

;(async () => {
  const exchange = new ccxt.bybit({
    apiKey: 'AAA',
    secret: 'sss',
    verbose: false, // set to true to see more debugging output
    timeout: 60000,
    enableRateLimit: true // add this
  })

  while (true) {
    try {
      await exchange.loadMarkets()
      break
    } catch (e) {
      if (e instanceof ccxt.RequestTimeout) { console.log(exchange.iso8601(Date.now()), e.constructor.name, e.message) }
    }
  }

  try {
    const response = await exchange.userGetLeverage()
    console.log(response)
  } catch (e) {
    console.log(exchange.iso8601(Date.now()), e.constructor.name, e.message)
    console.log('Failed')
  }
})()
```

```
2020-04-12T07:46:54.401Z ExchangeNotAvailable bybit GET https://api.bybit.comleverage/user/leverage?api_key=AAA&recvWindow=5000&timestamp=1586677604385&sign=84bc9361f7368efb9800ef445d2dee30be6c2bd9cd8455f645e9f0a0dc27413a system request to https://api.bybit.comleverage/user/leverage?api_key=AAA&recvWindow=5000&timestamp=1586677604385&sign=84bc9361f7368efb9800ef445d2dee30be6c2bd9cd8455f645e9f0a0dc27413a failed, reason: getaddrinfo ENOTFOUND api.bybit.comleverage
Failed
```

**Comment by kroitor** [comment]
@tradercgt fixed in CCXT 1.26.9+. We will be happy if you report back whether it works on your side or not. Thank you for the feedback!

**Comment by tradercgt** [comment]
now it works, thank you!
